includes/class-email-notifications.php

```diff
@@ -150,8 +150,10 @@ public function is_active( $case ) {
 
 		$option = $options[$case];
 
-		return boolval( wpas_get_option( $option, false ) );
-
+		/* Replace the valueless tags array by the new one */
+		return boolval( apply_filters( 'wpas_email_notifications_case_is_active', wpas_get_option( $option, false ) ));
```

Comment:
This is a good idea. One thing is missing, though. You need to pass the `$case` var as a parameter of this filter. Otherwise, someone hooking into it won't be able to target a specific case. The filter should be:

`
return (bool) apply_filters( 'wpas_email_notifications_case_is_active', wpas_get_option( $option, false ), $case );
`

Let's also typecast the result instead of using `boolval()` which isn't compatible with older versions of PHP. I used to use `boolval()` (there are probably more instanced throughout the code) but I believe typecasting is cleaner.